Makefile

```diff
@@ -25,7 +25,7 @@ test:
 
 docker:
 	@echo 'Building image $(IMAGE)...'
-	docker build --no-cache -t $(IMAGE) .
+	docker buildx build --output=type=docker --platform linux/arm64 --no-cache -t $(IMAGE) .
```

Comment:
does it mean `make docker` will make arm64 image always?

Reply:
I can make default to `go env GOARCH` (builder's machine architecture)

Reply:
sounds good to me!